File: modules/AcroStack.OpenIddictManagement/Dtos/UpdateOpenIddictScopeDto.cs
using System.Collections.Generic;

namespace AcroStack.OpenIddictManagement;

/// <summary>
/// DTO for updating an existing OpenIddict scope. <c>Name</c> is
/// intentionally omitted — changing it after creation is not supported.
/// </summary>
public class UpdateOpenIddictScopeDto
{
    public string? DisplayName { get; set; }

    public string? Description { get; set; }

    public List<string> Resources { get; set; } = new();
}

File: react/src/api/models/acroStack/openIddictManagement/UpdateOpenIddictScopeDto.ts
/* oxlint-disable */

/**
 * @description DTO for updating an existing OpenIddict scope. `Name` is\r\nintentionally omitted — changing it after creation is not supported.
 * @type object
 */
export type AcroStackOpenIddictManagementUpdateOpenIddictScopeDto = {
  displayName?: string | null;
  description?: string | null;
  resources?: string[] | null;
};
